bench/workgen/workgen_int.h

```diff
@@ -32,9 +32,12 @@
 #include <set>
 #ifndef SWIG
 extern "C" {
+#include <unistd.h>
 #include "workgen_func.h"
+#include <math.h>
 }
 #endif
+#include "workgen_time.h"
```

Comment:
I think the include of `"workgen_time.h"` should be within the `#ifndef SWIG`.  Otherwise (it think??) SWIG will include it and create interfaces for everything in `workgen_time.h`, we don't want that.  To check this, you might compare the difference between generated files like: `bench/workgen/workgen_wrap.cxx` and `bench/workgen/workgen/workgen.py`.  For each, compare what it looked like after a build without your changes, vs to after a build with your changes.  We'd only expect to see the new _user visible_ interfaces you've made.

Reply:
I checked the difference of generated files before and after my changes and only the user-visible interfaces are available. `#include "workgen_time.h"` should be outside `#ifndef SWIG`.

Reply:
Fixed.